src/Stratis.Bitcoin/Connection/ConnectionManagerBehavior.cs

```diff
@@ -85,7 +85,11 @@ private async Task OnStateChangedAsync(INetworkPeer peer, NetworkPeerState oldSt
 
                 if ((peer.State == NetworkPeerState.Failed) || (peer.State == NetworkPeerState.Offline))
                 {
-                    this.infoLogger.LogInformation("Peer '{0}' offline, reason: '{1}'.{2}", peer.RemoteSocketEndpoint, peer.DisconnectReason?.Reason ?? "unknown", peer.DisconnectReason?.Exception?.Message ?? string.Empty);
+                    // We only want to output information about connections to the user that actually meant something. Discovery peers are trivial.
+                    if (!peer.IsForPeerDiscovery)
```

Comment:
@codingupastorm I had a look at the code again and it seems that we do in fact clear all the behaviors from the peer before we connect. So IMO this beahvior (ConnectionManagerBehavior) will never be hit:

```
        {
            await peersToDiscover.ForEachAsync(5, this.nodeLifetime.ApplicationStopping, async (endPoint, cancellation) =>
            {
                using (CancellationTokenSource connectTokenSource = CancellationTokenSource.CreateLinkedTokenSource(cancellation))
                {
                    this.logger.LogDebug("Attempting to discover from : '{0}'", endPoint);

                    connectTokenSource.CancelAfter(TimeSpan.FromSeconds(5));

                    INetworkPeer networkPeer = null;

                    try
                    {
                        NetworkPeerConnectionParameters clonedParameters = this.currentParameters.Clone();
                        clonedParameters.ConnectCancellation = connectTokenSource.Token;

                        PeerAddressManagerBehaviour addressManagerBehaviour = clonedParameters.TemplateBehaviors.OfType<PeerAddressManagerBehaviour>().FirstOrDefault();
                        clonedParameters.TemplateBehaviors.Clear();
                        clonedParameters.TemplateBehaviors.Add(addressManagerBehaviour);

``` 

We only actually add the peer address manager's behavior.

You can test this by commenting out all the peer connectors from the code and then JUST running peer discovery to see if we hit this behavior on disconnection.